.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Chat/OnYourData/DataSourceFieldMappings.cs

```diff
@@ -21,9 +23,6 @@ public partial class DataSourceFieldMappings
     /// <see cref="AzureCosmosDBChatDataSource"/>, <see cref="ElasticsearchChatDataSource"/>, and
     /// <see cref="PineconeChatDataSource"/>.
     /// </para>
-    /// <para>
-    /// It is not applicable to types including: <see cref="AzureMachineLearningIndexChatDataSource"/>.
-    /// </para>
     /// </remarks>
     [CodeGenMember("UrlField")]
     public string UrlFieldName { get; init;}
```

Comment:
nit: I noticed that you're removing a few inits so I just wanted to point out the ones in this file in case you intended to fix these too as part of this PR.

Reply:
Thanks!! Not sure why I missed that file specifically, but fixed now. Per our chat, applying "get; set; for non-collections, get; for collections" for now -- we can figure out init use on collections later.